### PR TITLE
Add moreKafkaConfigs for KafkaZipkinFactory

### PR DESCRIPTION
**Pull request by xihw**
Resolve issue as mentioned in: https://github.com/smoketurner/dropwizard-zipkin/issues/42

### CODE REVIEW

File: zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/KafkaZipkinFactory.java [hunk 1]
```diff
@@ -43,6 +45,8 @@
     @NotEmpty
     private String topic = "zipkin";
 
+    private Map<String, String> moreKafkaConfigs = new LinkedHashMap<>();
```
Comment (jplock) [inline zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/KafkaZipkinFactory.java]: can we call this `overrides` instead?

**Comment by xihw** [comment]
@jplock Updated. 

Also do you know why Travis build fail? Doe it matter since I see it keeps happening recently ?

**Comment by jplock** [comment]
Thanks!

**Comment by jplock** [comment]
@xihw It's related to openjdk10 being available.  At one point I switched it from jdk9 to 10 when I guess things weren't ready at Travis.  Haven't had time to fix it yet.

**Comment by jplock** [comment]
Released in v1.3.2-2, just pushed it to Maven Central now.

**Pull request merged**